### DISCUSSION TITLE
`FastAPI.openapi` throws ValidationError, causing openapi docs page to crash

### DISCUSSION THREAD

**Discussion by HansBrende** [fastapi/fastapi#6537] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from pydantic import BaseModel, confloat

app = FastAPI()

LngLat = Tuple[confloat(ge=-180.0, le=180.0), confloat(ge=-90, le=90)]

class Viewport(BaseModel):
    sw: LngLat
    ne: LngLat

@app.get('/viewport', response_model=Viewport)
def get_viewport():
    return Viewport(sw=[-93, 36], ne=[-92, 37])
```

### Description

The example code I've provided causes the openapi docs page to crash, with the following error displayed:

> **Failed to load API definition.**
> **Errors
> Fetch error**
> Internal Server Error    /openapi.json

Due to the following stack trace:
```
  File "/Users/hansbrende/miniconda3/envs/facade-api/lib/python3.8/site-packages/fastapi/applications.py", line 161, in openapi
    return JSONResponse(self.openapi())
  File "/Users/hansbrende/miniconda3/envs/facade-api/lib/python3.8/site-packages/fastapi/applications.py", line 136, in openapi
    self.openapi_schema = get_openapi(
  File "/Users/hansbrende/miniconda3/envs/facade-api/lib/python3.8/site-packages/fastapi/openapi/utils.py", line 411, in get_openapi
    return jsonable_encoder(OpenAPI(**output), by_alias=True, exclude_none=True)  # type: ignore
  File "pydantic/main.py", line 406, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 3 validation errors for OpenAPI
components -> schemas -> Viewport -> properties -> sw -> items
  value is not a valid dict (type=type_error.dict)
components -> schemas -> Viewport -> properties -> ne -> items
  value is not a valid dict (type=type_error.dict)
components -> schemas -> Viewport -> $ref
  field required (type=value_error.missing)
  ```

### Operating System

macOS

### FastAPI Version

0.68.0

### Python Version

Python 3.8.10

**Comment by HansBrende** [comment]
Whoops, duplicate of #3665 it looks like.

**Comment by tiangolo** [comment]
Thanks for realizing it was a duplicate and closing it! You earned a cookie! :cookie:
